### PR TITLE
Fix build warning and merge some version together

### PR DESCRIPTION
**Pull request by ebarboni**
- Tried to use same version of Apache NetBeans artefacts for testing ${netbeans.version} instead of several hardcoded.
- Use plugin management a bit more. 
- Fix some tag related warning

As per my test I encounter #159 so this can be solved by excluding `asm ` from `maven-dependency-analyzer` from dependency and using it directly.

### CODE REVIEW

File: nbm-maven-plugin/pom.xml [hunk 1]
```diff
@@ -135,8 +135,18 @@ under the License.
...
+                    <groupId>org.ow2.asm</groupId>
+                    <artifactId>asm</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm</artifactId>
+            <version>9.7</version>
         </dependency>
```
Comment (mbien) [inline nbm-maven-plugin/pom.xml:149]: is the exclusion required? I thought a direct dependency would win over dependencies deeper down the tree.

**Review by mbien — APPROVED** [review]
looks good

File: nbm-maven-plugin/pom.xml (hunk 1 above)
Comment (mbien) [reply nbm-maven-plugin/pom.xml:149]: but you might be right, maven does sometimes downgrade versions

Comment (ebarboni) [reply nbm-maven-plugin/pom.xml:149]: With exclusion I would say I'm sure

Comment (mbien) [reply nbm-maven-plugin/pom.xml:149]: i was curious and checked since I always forget if the exclude is needed: version upgrade works without exclude

**Pull request merged**